sandbox/grommet-app/src/pages/layouts/kinds/HPEMarketing/MarketingBrochure.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useContext } from 'react';
+import {
+  Box,
+  Button,
+  Grid,
+  Image,
+  Heading,
+  PageContent,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import { LinkNext } from 'grommet-icons';
+import female from '/female.png';
+import clock from '/clock.svg';
+import performance from '/performance.svg';
+
+const ContentArea = ({ alt, image, desc, title }) => {
+  const breakpoint = useContext(ResponsiveContext);
+  console.log(breakpoint);
+  return (
+    <Box align="center" gap="medium">
+      <Box
+        width={breakpoint === 'medium' ? 'xxsmall' : 'xsmall'}
+        height={breakpoint === 'medium' ? 'xxsmall' : 'xsmall'}
+      >
+        <Image fit="contain" alt={alt} src={image} />
+      </Box>
+      <Heading margin="none" level={4}>
+        {title}
+      </Heading>
+      <Text>{desc}</Text>
+    </Box>
+  );
+};
+
+export const MarketingBrochure = () => {
+  const breakpoint = useContext(ResponsiveContext);
+
+  const columns = ['xsmall', 'small'].includes(breakpoint)
+    ? ['auto']
+    : {
+        count: 3,
+        size: 'auto',
+      };
+  const rows = ['auto'];
+  const gap = {
+    xsamll: 'medium',
+    small: 'medium',
+    medium: 'medium',
+    large: 'large',
+    xlarge: 'large',
+  };
+
+  return (
+    <PageContent
+      pad={{ vertical: 'xlarge', horizontal: 'xlarge' }}
+      background={{ fill: 'horizontal', color: 'green', dark: true }}
+      fill="horizontal"
+    >
+      <Box gap="large">
+        <Grid
+          alignSelf="center"
+          columns={columns}
+          rows={rows}
+          gap={gap[breakpoint]}
+        >
+          <ContentArea
+            image={female}
+            alt=""
+            title="People"
+            desc="Work with experts and develop new skill sets"
+          />
+          <ContentArea
+            image={clock}
+            alt=""
+            title="Technology"
+            desc="Accelerate AI deployments and protect from risks"
+          />
+          <ContentArea
+            image={performance}
+            title="Economics"
+            desc="Optimize AI costs long term"
+            alt=""
+          />
+        </Grid>
+        <Button
+          size="large"
+          icon={<LinkNext />}
+          label="Read the brochure"
+          alignSelf="center"
```

Comment:
```suggestion
          alignSelf="center"
          reverse
          secondary
```

Reply:
Ill look

Reply:
<img width="393" alt="Screenshot 2025-03-04 at 12 38 00 PM" src="https://github.com/user-attachments/assets/f4d8fdd4-40f1-49b4-a7ff-514bb9d09c6e" />


is that what secondary is suppose to look like 🤔

Reply:
added these as well as added a comment the `background-contrast` doesnt work well